armi/utils/tests/test_hexagon.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test hexagon tools."""
+import unittest
+
+from armi.utils import hexagon
+import math
```

Comment:
This whole file is great and I love that you did this.

But, just FYI, if you look at the rest of the files in ARMI you will see we put the imports at the top of the file, in a certain order:

```python
import "standard library things"

import "third party library things"

import "local things"
```

Does that make sense?

So, in this case, the imports here would look like:

```python
import math
import unittest

from armi.utils import hexagon
```